index.bs

```diff
@@ -3539,7 +3539,7 @@ lt="WritableStreamDefaultWriter(stream)">new WritableStreamDefaultWriter(<var>st
   1. Return *this*.[[closedPromise]].
 </emu-alg>
 
-<h5 id="default-writer-desiredSize" attribute for="WritableStreamDefaultWriter" lt="desiredSize">get desiredSize</h5>
+<h5 id="default-writer-desired-size" attribute for="WritableStreamDefaultWriter" lt="desiredSize">get desiredSize</h5>
```

Comment:
I changed this to have consistent spelling with the other IDs and couldn’t find any other references to it. I am assuming I’m not breaking anything herE?